projects/fmcdaq2/src/app/fmcdaq2.c

```diff
@@ -990,9 +990,9 @@ static int fmcdaq2_setup(struct fmcdaq2_dev *dev,
 	if (status != 0)
 		return status;
 
-	status = ad9144_setup(&dev->ad9144_device, &dev_init->ad9144_param);
+	status = ad9144_setup_legacy(&dev->ad9144_device, &dev_init->ad9144_param);
```

Comment:
why a legacy ? aren't we supposed to use the new jesd setup ?

Reply:
This function was renamed to "_legacy" because the new JESD FSM implementation uses a new setup function with the same name . This renamed one is the old setup function.